java-components/cli/src/test/java/io/github/redhatappstudio/jvmbuild/cli/rebuilt/RebuiltDownloadCommandTest.java

```diff
@@ -31,11 +31,9 @@
 @QuarkusTest
 public class RebuiltDownloadCommandTest {
 
-    @ConfigProperty(name = "store.rebuilt.registry")
-    String registry;
```

Comment:
The config injection here meant that all tests expected this to be set. Moving to programatic config resolution removed the need to set this value for other tests.